### PR TITLE
bug/eugene/datless/fix-local-tests-for-py3

### PR DESCRIPTION
**Pull request by eugsokolov**
this is the python3 way

### CODE REVIEW

File: fabfile/build.py [hunk 1]
```diff
@@ -39,8 +39,8 @@ def python_environment(
         os.makedirs(base_dir)
 
     with lcd(base_dir):
-        local("virtualenv {}".format(dir_name))
-        local("{}/pip install --upgrade pip".format(bin_dir))
-        local("{}/pip install -r {}".format(bin_dir, path_to_requirements))
+        local("python3 -m venv {}".format(dir_name))
```
Comment (reyna72) [inline fabfile/build.py]: should we update the requirements.txt too?

**Comment by eugsokolov** [comment]
retest this please

**Comment by eugsokolov** [comment]
retest this please

**Comment by eugsokolov** [comment]
retest this please

**Comment by eugsokolov** [comment]
retest this please

**Comment by eugsokolov** [comment]
retest this please

**Comment by eugsokolov** [comment]
retest this please

**Comment by eugsokolov** [comment]
retest this please

**Comment by eugsokolov** [comment]
retest this please

File: fabfile/build.py (hunk 1 above)
Comment (eugsokolov) [reply fabfile/build.py]: we can

**Pull request merged**